docusaurus/docs/iOS/swiftui/chat-channel-components/message-list.md

```diff
@@ -302,6 +303,12 @@ let utils = Utils(messageListConfig: messageListConfig)
 streamChat = StreamChat(chatClient: chatClient, utils: utils)
 ```
 
+### embedInNavigationView
```

Comment:
This would need to be moved to the docs about the channel list view.

Reply:
Updated documentation  to reflect recent changes